help_to_heat/settings.py

```diff
@@ -258,7 +258,7 @@ def show_toolbar(request):
 
 TOTP_ISSUER = "Help to Heat Supplier Portal"
 
-CSRF_TRUSTED_ORIGINS = [BASE_URL]
+CSRF_TRUSTED_ORIGINS = [BASE_URL.rstrip("/")]
```

Comment:
Would quite like a comment explaining why we need to trim it, if it's not obvious